compiler/luci/partition/src/PartitionMerge.cpp

```diff
@@ -58,6 +58,9 @@ bool is_input_same(const luci::PGroup *pgroup, const luci::PGroups *pgroups)
     //         we need to clone this CircleConst for each graph of the group.
     if (dynamic_cast<const luci::CircleConst *>(input) != nullptr)
       continue;
+    // Skip for OutputExclude
+    if (dynamic_cast<const luci::CircleOutputExclude *>(input) != nullptr)
+      continue;
```

Comment:
@chunseoklee , can you check with adding this patch (above 3 lines including the comment) and check your model?

Reply:
It works on my local !